flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/enumerator/subscriber/KafkaSubscriberTest.java

```diff
@@ -45,17 +45,19 @@ public class KafkaSubscriberTest {
     private static final TopicPartition NON_EXISTING_TOPIC = new TopicPartition("removed", 0);
     private static AdminClient adminClient;
 
-    @BeforeClass
+    @BeforeAll
     public static void setup() throws Throwable {
         KafkaSourceTestEnv.setup();
         KafkaSourceTestEnv.createTestTopic(TOPIC1);
         KafkaSourceTestEnv.createTestTopic(TOPIC2);
         adminClient = KafkaSourceTestEnv.getAdminClient();
     }
 
-    @AfterClass
+    @AfterAll
     public static void tearDown() throws Exception {
```

Comment:
```suggestion
    static void tearDown() throws Exception {
```
could be package private